package.json

```diff
@@ -24,14 +24,18 @@
     "dev": "lerna run --parallel --scope @firebase/* --scope firebase dev",
     "build": "lerna run --scope @firebase/* --scope firebase build",
     "build:changed": "ts-node-script scripts/ci-test/build_changed.ts",
+    "build:ci:auth": "lerna run build:deps --scope '@firebase/auth*'",
+    "build:ci:firestore": "lerna run build:deps --scope '@firebase/firestore*'",
     "release:prepare": "lerna run --scope @firebase/* add-compat-overloads && lerna run --scope @firebase/* typings:public",
     "link:packages": "lerna exec --scope @firebase/* --scope firebase -- yarn link",
     "stage:packages": "./scripts/prepublish.sh",
     "repl": "node tools/repl.js",
     "release": "ts-node-script scripts/release/cli.ts",
     "pretest": "node tools/pretest.js",
     "test": "lerna run --ignore firebase-messaging-integration-test --concurrency 4 --stream test",
-    "test:ci": "lerna run --ignore firebase-messaging-integration-test --concurrency 4 test:ci",
+    "test:ci": "lerna run --ignore '{firebase-messaging-integration-test,@firebase/auth*,@firebase/firestore*}' --concurrency 4 test:ci",
```

Comment:
I know we won't be using the old command in CI anymore but since you can't leave comments in a JSON file, the best context for why these packages are being excluded is in the workflow, so I think it might be best to put all these specifics in the workflow by calling lerna directly as mentioned in the other comment. Maybe we can get rid of this script or just leave it in just in case we ever go back to running all the tests at once.

Reply:
Done.